test/framework/src/main/java/org/elasticsearch/test/rest/yaml/ClientYamlTestClient.java

```diff
@@ -57,7 +57,11 @@
 public class ClientYamlTestClient {
     private static final ESLogger logger = Loggers.getLogger(ClientYamlTestClient.class);
     //query_string params that don't need to be declared in the spec, they are supported by default
-    private static final Set<String> ALWAYS_ACCEPTED_QUERY_STRING_PARAMS = Sets.newHashSet("pretty", "source", "filter_path");
+    private static final Set<String> ALWAYS_ACCEPTED_QUERY_STRING_PARAMS = Sets.newHashSet("pretty",
+                                                                                            "source",
+                                                                                            "filter_path",
+                                                                                            "filter_path_include",
+                                                                                            "filter_path_exclude");
```

Comment:
I agree, but I finally removed the `filter_path_excludes` and  `filter_path_includes` in favor of a `+/-` support using the current `filter_path` parameter, as Clinton suggested.